src/mentions/webmentions.json

```diff
@@ -0,0 +1,4 @@
+{
+  "lastFetched": "2022-06-16T20:33:08.486Z",
+  "children": []
+}
```

Comment:
I'm open to other ideas, but I don't think it's worth having a cache if we don't commit it, right? The whole point is that it's caching mentions (in `children`, once we have any) so that we don't have to re-download them every time.

Reply:
probably? do you want to look into it? Netlify has caching capabilities, and there are some netlify plugins to help, but I haven't been able to figure it out on my personal site.

Reply:
Sure, I'll add a story. https://trello.com/c/Oi2HHuLj